### ISSUE TITLE
RN 0.62 Could not resolve project :navigation-react-native

### ISSUE THREAD

**Issue by SatyaFariz** [grahammendick/navigation#366]
I've already carefully follow the [upgrade helper tool](https://react-native-community.github.io/upgrade-helper/?from=0.61.5&to=0.62.0) but got an error like this:

<img width="637" alt="Capture" src="https://user-images.githubusercontent.com/25284201/78283864-ad12b180-7548-11ea-93c5-1d1d4fa93e96.PNG">

**Comment by grahammendick** [comment]
Thanks for letting me know.

Looks like the android build has changed a bit (I might’ve had it wrong all along and just got away with it pre < 62?). I’ve got it working locally but I need to do some more testing, e.g., ensure it’s backward compatible.

I don’t want to hold you up so here are the two changes (I can’t guarantee that this is what the final changes will look like). 

1. Go to the `node_modules/navigation-react-native/android` folder and paste the following into build.gradle
```
// Top-level build file where you can add configuration options common to all sub-projects/modules.

buildscript {

    repositories {
        google()
        jcenter()
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:3.2.0'


        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
}

apply plugin: 'com.android.library'

def safeExtGet(prop, fallback) {
    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
}

android {
    compileSdkVersion safeExtGet("compileSdkVersion", 28)
    buildToolsVersion safeExtGet("buildToolsVersion", "28.0.3")

    defaultConfig {
        minSdkVersion safeExtGet("minSdkVersion", 16)
        targetSdkVersion safeExtGet("targetSdkVersion", 28)
        versionCode 1
        versionName "1.0"
    }

    lintOptions {
        abortOnError false
    }
}

repositories {
    mavenCentral()
}

dependencies {
    implementation "com.facebook.react:react-native:+"
    implementation 'com.google.android.material:material:1.0.0'
}
```
2. Move the `node_modules/navigation-react-native/android/app/src` folder and its contents up one level, into the `node_modules/navigation-react-native/android` folder

**Comment by grahammendick** [comment]
Just released the fix in `navigation-react-native 6.7.1`. [I thanked you](https://github.com/grahammendick/navigation/releases/tag/v6.7.1-NavigationReactNative) in the release notes
